compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/window/Application.desktop.kt

```diff
@@ -240,6 +241,7 @@ suspend fun awaitApplication(
 /**
  * Scope used by [application], [awaitApplication], [launchApplication]
  */
+@Stable
```

Comment:
Is it correct to put @Stable on interface? It seems to be suitable for implementation per https://developer.android.com/reference/kotlin/androidx/compose/runtime/Stable

Reply:
The docs say that is allowed:
```
When applied to a class or an interface, [Stable] indicates that the following must be true
```

Also, a lot of interfaces in Compose marked by this annotation.